package.json

```diff
@@ -128,34 +140,22 @@
     "@types/node": "^14.14.31",
     "@typescript-eslint/eslint-plugin": "^5.58.0",
     "@typescript-eslint/parser": "^5.58.0",
-    "acorn": "^8.8.2",
     "agadoo": "^3.0.0",
-    "aria-query": "^5.1.3",
-    "axobject-query": "^3.1.1",
-    "code-red": "^1.0.0",
-    "css-tree": "^2.3.1",
     "eslint": "^8.35.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-svelte3": "^4.0.0",
-    "estree-walker": "^3.0.3",
-    "is-reference": "^3.0.1",
     "jsdom": "^21.1.1",
     "kleur": "^4.1.5",
-    "locate-character": "^2.0.5",
-    "magic-string": "^0.30.0",
     "mocha": "^10.2.0",
-    "periscopic": "^3.1.0",
     "prettier": "^2.8.8",
     "prettier-plugin-svelte": "^2.10.0",
     "puppeteer": "^19.8.5",
     "rollup": "^3.20.2",
-    "source-map": "^0.7.4",
```

Comment:
`source-map` is used by the tests so it should remain in devDependencies.

Reply:
fixed